benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -274,16 +274,16 @@ msgstr ""
 msgid "core.pages.help.foss.link"
 msgstr "GitHub"
 
+msgid "core.buttons.previous_page"
+msgstr "Previous Page"
+
```

Comment:
I'm wondering why the script moved all these other entries around.

Reply:
It's so mysterious. It always moves these.